pgoapi/pgoapi.py

```diff
@@ -308,6 +358,19 @@ def spin_all_forts_visible(self):
     def return_to_start(self):
         self.set_position(*self._origPosF)
 
+    def walk_to_fort(self, fort_data, directly=False):
+        fort = fort_data[0]
+        self.log.info("Walking to fort at  http://maps.google.com/maps?q=%s,%s", fort['latitude'],
+                        fort['longitude'])
+        if directly is True:
```

Comment:
looks like quite a lot of recursion going on here, in walk to fort if directly is false, it calls Walk_to which in turn calls,     spin nearest fort, then if the fort is not close enough , it calls walk to again.. i am just worried about this.  The thing you put spin_nearest_fort within 40m if not then walk 100 meters to it, (use google false) take fort, and then come back where it started 

if for not in 40 m but within 100 meter
cur_pos = pos
walk_to(for_pos)
walk_to(cur_pos) 
and continue